src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -446,7 +451,7 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = (props) => {
           {templateID && BACKGROUND_IMG_TEMPLATES.includes(templateID) && (
             <>
               <h4 className="italic" style={{ fontSize: "20px" }}>
-                {`Choose the background for your Theme Camp`}
+                {`Choose the background for your ${backgroundTextByVenue[templateID]}`}
```

Comment:
add default fallback if `backgroundTextByVenue` doesn't have a match

something like:
```suggestion
                {`Choose the background for your ${backgroundTextByVenue[templateID] ?? 'FOO DEFAULT'}`}
```